pydeseq2/utils.py

```diff
@@ -941,18 +948,25 @@ def nbinomGLM(
     converged: bool
         Whether L-BFGS-B converged.
     """
-    beta_init = np.array([0.1, -0.1])
-    no_shrink_index = 1 - shrink_index
+
+    p = design_matrix.shape[-1]
```

Comment:
here again I would suggest changing the name, up to teste

Reply:
Thanks, done in [e536451](https://github.com/owkin/PyDESeq2/pull/32/commits/e5364518043f4edf9dccaf314ad8020bc7ba2b87)